.conda/sleap_activate.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+# Help CUDA find GPUs!
+export LD_LIBRARY_PATH=$CONDA_PREFIX/lib:$LD_LIBRARY_PATH
```

Comment:
Are you sure this will work? I thought the cuda binaries were stored in the `$CONDA_PREFIX/envs/{env_name}/lib` or something.

Like, one blocker for upgrading to newer tensorflow was that we couldn't dynamically set this since users would have different env names.

If it's really in `$CONDA_PREFIX/lib` and it's not just hardcoding the path from the computer you built in (would Jinja do this?), then that's awesome and would allow us to move up to tf 2.9 I think!

If it _doesn't_ work, then it might be ok to just include the extra line that people have to run in the install docs?

Reply:
```bash
(nb4) talmolab@talmolab-01-ubuntu:~/sleap-estimates-animal-poses/pull-requests/sleap$ echo $CONDA_PREFIX
/home/talmolab/micromamba/envs/nb4
```

!!!

Reply:
>If it's really in $CONDA_PREFIX/lib and it's not just hardcoding the path from the computer you built in (would Jinja do this?), then that's awesome and would allow us to move up to tf 2.9 I think!

I think that might be the point of using `cp` now that you mention it (instead of `echo ... >> ...`).